### ISSUE TITLE
Somethin errors

### ISSUE THREAD

**Issue by dwisiswant0** [Manisso/fsociety#38]
I got this err msg bellow when I choose **Private Web Hacking** > **Ports Scan (range of ports)**.

```bash
Traceback (most recent call last):
  File "/home/dw1/.fsociety/fsociety.py", line 2077, in <module>
    fsociety()
  File "/home/dw1/.fsociety/fsociety.py", line 148, in __init__
    wirelessTestingMenu()
  File "/home/dw1/.fsociety/fsociety.py", line 820, in __init__
    fsociety()
  File "/home/dw1/.fsociety/fsociety.py", line 156, in __init__
    privateWebHacking()
  File "/home/dw1/.fsociety/fsociety.py", line 319, in __init__
    Fscan(target)
  File "/home/dw1/.fsociety/fsociety.py", line 1374, in __init__
    self.portScanner(1, ran)
AttributeError: Fscan instance has no attribute 'portScanner'
```

Please resolve this. I think this tools is so fucking awesome!!

**Comment by Jinovas** [comment]
was checking out this tool and also getting similar error. I've installed and used inurlbr before without error. 

sh: 1: ./inurlbr.php: not found
Traceback (most recent call last):
  File "fsociety.py", line 2077, in <module>
    fsociety()
  File "fsociety.py", line 158, in __init__
    postExploitationMenu()
  File "fsociety.py", line 351, in __init__
    fsociety()
  File "fsociety.py", line 156, in __init__
    privateWebHacking()
  File "fsociety.py", line 319, in __init__
    Fscan(target)
  File "fsociety.py", line 1382, in __init__
    fsociety()
  File "fsociety.py", line 154, in __init__
    webHackingMenu()
  File "fsociety.py", line 275, in __init__
    ifinurl()
  File "fsociety.py", line 1042, in ifinurl
    inurl()
  File "fsociety.py", line 1097, in inurl
    webHackingMenu.completed("InurlBR")
TypeError: unbound method completed() must be called with webHackingMenu instance as first argument (got str instance instead)

**Comment by vasanth3045** [comment]
Fix available in #49.
Pls review & merge it
